docker/compose/local-dev-compose.yml

```diff
@@ -26,17 +26,29 @@ services:
   filer:
     image: chrislusf/seaweedfs:local
     ports:
-      - 8111:8111
       - 8888:8888
       - 18888:18888
-    command: '-v=1 filer -ip.bind=0.0.0.0 -master="master:9333" -iam -iam.ip=filer'
+    command: '-v=1 filer -ip.bind=0.0.0.0 -master="master:9333"'
     depends_on:
       - master
       - volume
     volumes:
       - ./tls:/etc/seaweedfs/tls
     env_file:
       - ${ENV_FILE:-dev.env}
+
+  iam:
+    image: chrislusf/seaweedfs:local
+    ports:
+      - 8111:8111
+    command: '-v=1 iam -filer="filer:8888" -master="master:9333"'
+    depends_on:
+      - master
+      - volume
+      - filer
+    volumes:
+      - ./tls:/etc/seaweedfs/tls
+
```

Comment:
IMO it's beneficial to have `iam` be separated, so that it's logs can be viewed separately using `docker container logs seaweedfs-iam-1 --follow`.

Feel free to disagree and I'll revert it back